tools/Equinox.Tool/Program.fs

```diff
@@ -506,7 +506,7 @@ module Dump =
                  with e -> log.Warning(e, "UTF-8 Parse failure - use --Blobs option to inhibit"); reraise()
         let readStream (streamName : FsCodec.StreamName) = async {
```

Comment:
Should this be `readTarget` and `target`?

Reply:
no, it's definitely reading the stream, and the target is a StreamName at this point, having left its primitive roots behind (but I named it `dumpEvents`)

Reply:
More of a facetious comment from my end, trying to solidify where to draw the line between the stream and the target